src/movies.js

```diff
@@ -1,13 +1,34 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(movies) {
+    const myArr = movies.filter(function (director) {
+      });
+      return myArr
+      console.log(myArr);
```

Comment:
The piece of code below this line will never be executed, because we have a return statement on line 7